python/pyspark/streaming/tests.py

```diff
@@ -611,12 +611,15 @@ class CheckpointTests(unittest.TestCase):
     @staticmethod
     def tearDownClass():
         # Clean up in the JVM just in case there has been some issues in Python API
-        jStreamingContextOption = StreamingContext._jvm.SparkContext.getActive()
+        jStreamingContextOption = \
+            SparkContext._jvm.org.apache.spark.streaming.StreamingContext.getActive()
         if jStreamingContextOption.nonEmpty():
             jStreamingContextOption.get().stop()
-        jSparkContextOption = SparkContext._jvm.SparkContext.get()
```

Comment:
`SparkContext` doesn't provide a method to get the active one. So just removed it.